<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGraduatedSymbolRendererV2Widget</class>
 <widget class="QWidget" name="QgsGraduatedSymbolRendererV2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>341</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Column:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cboGraduatedColumn"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Symbol:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnChangeGraduatedSymbol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>change</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Classes:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="spinGraduatedClasses">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Color ramp:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboGraduatedColorRamp"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="cboGraduatedMode">
       <item>
        <property name="text">
         <string>Equal Interval</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quantile</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="viewGraduated">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <widget class="QPushButton" name="btnGraduatedClassify">
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGraduatedDelete">
       <property name="text">
        <string>Delete class</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboGraduatedColumn</tabstop>
  <tabstop>btnChangeGraduatedSymbol</tabstop>
  <tabstop>cboGraduatedColorRamp</tabstop>
  <tabstop>spinGraduatedClasses</tabstop>
  <tabstop>cboGraduatedMode</tabstop>
  <tabstop>viewGraduated</tabstop>
  <tabstop>btnGraduatedClassify</tabstop>
  <tabstop>btnGraduatedDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
